src/uabytestring.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * uabytestring.cpp
+ *
+ *  Created on: Jan 26, 2017
+ *      Author: pnikiel
+ */
+
+#include <uabytestring.h>
+
+#include <open62541_compat_common.h>
+
+UaByteString::UaByteString ()
+{
+	m_impl = UA_ByteString_new();
+	if (!m_impl)
+		throw alloc_error();
+	UA_ByteString_init(m_impl);
```

Comment:
should we set the m_impl data and length members here, or we trust UA_ByteString* does the right thing?